### ISSUE TITLE
4.24/4.25 cannot view JMX beans in dashboard

### ISSUE THREAD

**Issue by space88man** [spring-projects/sts4#1350]
**Describe the bug**
OS: Fedora 40, x86_64
Java: using the jre inside STS

Since 4.24.0 with actuator/JMX/beans enabled, scraping beans doesn't show in STS dashboard. The endpoint can be scraped with jconsole.

Working 4.23.1 (can scrape beans):

![Screenshot from 2024-09-11 19-53-30](https://github.com/user-attachments/assets/faa89190-94ae-4e39-a2b3-e4b200bca79c)

Not working 4.24.0/4.25 nightly (cannot scrape beans):

![Screenshot from 2024-09-11 19-55-37](https://github.com/user-attachments/assets/e8a4a415-fc67-42af-a3e4-69c2c3290afe)

**To Reproduce**

- Create a Spring starter with Web/Actuator
- Enable JMX in Run config

**Additional Information**

- 4.25 nightly build with Eclipse 4.31 works(!)
- 4.25 nightly build with Eclipse 4.32 works(!)

**Sample**

[demo-1.zip](https://github.com/user-attachments/files/16963209/demo-1.zip)

**Comment by space88man** [comment]
Vanilla 4.24.0-RELEASE (with no p2 updates) is still working:
```
com.sun.management.jmxremote.authenticate=false
com.sun.management.jmxremote.ssl=false
com.sun.management.jmxremote=
spring.jmx.enabled=true
management.endpoints.jmx.exposure.include=*
com.sun.management.jmxremote.port=40387
```

**Comment by BoykoAlex** [comment]
Fixed with 30abaec4d6e2bf6d2fe09c500e388edca3e1ee25

**Comment by space88man** [comment]
> Fixed with [30abaec](https://github.com/spring-projects/sts4/commit/30abaec4d6e2bf6d2fe09c500e388edca3e1ee25)
I have built and overridden just that class into `sts-4.25.0.RELEASE/` and can confirm that my issue is fixed - the  beans pane now renders correctly. Thank you!
